src/views/layouts/layout.hbs

```diff
@@ -7,6 +7,8 @@
     <body>
         {{> header}}
         {{{content}}}
-        {{> footer}}
+        <div class="container-fluid text-center">
```

Comment:
The reason for the difference is due to the home page not using the existing layout-file (and no header-partial), but still re-using the footer-partial which is present on all pages. And, the header-container is wrapped in a nav-bar which is different than the footer where the container is the parent-element. A feature which did not make the final-cut was having the footer on non-home page views left-justified. However, letting the parent-view control the placement of the footer seemed like a good idea, so I kept the related changes as-is.